src/main/java/frc/robot/commands/DeployMailbox.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.RobotContainer;
+
+public class DeployMailbox extends ParallelCommandGroup {
+  /** Deploys the mailbox we use to output gamepieces */
+  public DeployMailbox() {
+    // Add your commands in the addCommands() call, e.g.
+    // addCommands(new FooCommand(), new BarCommand());
+    addCommands(RobotContainer.deployPneumatics, RobotContainer.runBelts);
```

Comment:
Okay, I'm sorry, I super lied to you. Instead of doing `RobotContainer.deployPneumatics`, create yourself a `new DeployPneumatics`. This is because you have to create a new command object for each command group that that command is a part of, otherwise the command scheduler gets mad and throws an error